### PR TITLE
Task 1: add alternative solution  FizzBuzz

### PR DESCRIPTION
**Pull request by Dariastep**
Task 2: add alternative solution  FizzBuzz

### CODE REVIEW

**Review by Dariastep — COMMENTED** [review]
Super!

**Comment by just-fy** [comment]
Waiting for the solution!

**Comment by Dariastep** [comment]
Added an alternatiove task solution with the help of the ternary operator.
To be rewieved by @just-fy

File: FizzBazz.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
-const fizzBuzz = () => {
-    console.log('text');
+function fizzBuzz() {
```
Comment (just-fy) [inline FizzBazz.js:1]: Why have you decided to use `function` declaration instead of arrow function?

File: FizzBazz.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
+     /* add alternative solution (ternary operator) */
```
Comment (just-fy) [inline FizzBazz.js:2]: No need in comment anymore, I think we will leave this solution

File: FizzBazz.js [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
+    console.log((i % 3 === 0 && i % 5 === 0) ? 'fizzBuzz'
+    : (i % 5 === 0) ? 'buzz'
+    : (i % 3 === 0) ? 'fizz'
+    : i);
     }
+     
 }
-fizzBuzz();
\ No newline at end of file
+fizzBuzz();
```
Comment (just-fy) [inline FizzBazz.js:11]:
```suggestion
fizzBuzz();
```

**Review by just-fy — COMMENTED** [review]
LGTM, left nit comments 🙂

File: FizzBazz.js (hunk 2 above)
Comment (Dariastep) [reply FizzBazz.js:2]: OK

**Review by just-fy — APPROVED** [review]

**Pull request merged**